modules/nextflow/src/main/groovy/nextflow/processor/Architecture.groovy

```diff
@@ -45,7 +45,7 @@ class Architecture {
  * [alternate example: 'arch linux/arm/v8', where platform = 'linux' and arch = 'arm/v8']
  */
     // used in Nextflow
-    final String dockerArch
+    final String dockerPlat
```

Comment:
Think this should stay `dockerArch` or at least postpone the rename to a later stage

Reply:
Just notice that across the codebase in NF and Wave, the corresponding value is typically referred to as `containerPlatform` or `platform` ; hence my change here. 

Does it make more sense now?

Reply:
yeah, but makes more difficult to track the real changes here

Reply:
OK, reverting here, opened a dedicated PR #4706